src/components/TemplatesPage/TemplatesPage.tsx

```diff
@@ -0,0 +1,64 @@
+import { useCallback } from 'react'
+import { Button, Page, Icon } from 'decentraland-ui'
+import { locations } from 'routing/locations'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import Footer from 'components/Footer'
+import Navbar from 'components/Navbar'
+import { SceneCard } from 'components/SceneCard'
+import Navigation from 'components/Navigation'
+import { NavigationTab } from 'components/Navigation/Navigation.types'
+import { Props } from './TemplatesPage.types'
+import templates from './templates.json'
+import styles from './TemplatesPage.module.css'
+
+export default function TemplatesPage({ onNavigate }: Props) {
```

Comment:
same as the previous component, shouldn't we return a `React.FC` here to keep consistency?